src/tests/JIT/Regression/JitBlue/Runtime_45090/Runtime_45090.tt

```diff
@@ -0,0 +1,89 @@
+<#@ template debug="false" hostspecific="true" language="C#" #>
+<#@ output extension=".il" #>
+<#@ assembly name="System.Core" #>
+<#@ import namespace="System.Collections.Generic" #>
+<#@ import namespace="System.Linq" #>
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Use the following commands to generate a corresponding IL file
+//   "%DevEnvDir%\TextTransform.exe" Runtime_45090.tt
+
+.assembly extern System.Runtime
+{
+}
+
+.assembly Runtime_45090
+{
+}
+
+.module Runtime_45090.dll
+
+<#
+  foreach(int structSizeInBytes in StructSizesInBytes)
+  {
+    string structName = StructName(structSizeInBytes);
+#>
+
+.class private explicit ansi sealed beforefieldinit Runtime_45090.<#= structName #>
```

Comment:
What is special about this test that it cannot be written in C#?

Reply:
It probably can. But it was simpler for me to express conditions for `AllocLocal` in order to have esp-based frame.

Reply:
I rewrote the test in C# - turned out that it was enough to make a method virtual to make inliner cooperate and not inline the method while keeping its frame esp based (I couldn't use `NoInlining` attribute since the method would have `JIT_FLAG_FRAMED` flag set and become ebp based).